tests/create_users_policies/main.tf

```
@@ -189,7 +185,7 @@ locals {
 }
 
 locals {
-  test_id = data.terraform_remote_state.prereq.outputs.random_string.result
+  test_id = data.terraform_remote_state.prereq.outputs.random_string
```

This is passing for me without these changes, in a local test run. I'm guessing they were from some earlier implementation and no longer relevant.

```suggestion
  test_id = data.terraform_remote_state.prereq.outputs.random_string.result
```

---

ok